PivotCalibration/Logic/vtkSlicerPivotCalibrationLogic.h

```diff
@@ -69,10 +69,14 @@ class VTK_SLICER_PIVOTCALIBRATION_MODULE_LOGIC_EXPORT vtkSlicerPivotCalibrationL
   // Returns with false on failure
   bool ComputeSpinCalibration( bool snapRotation = false ); //Note: The neede orientation protocol assumes that the shaft of the tool lies along the positive x-axis
 
+  // Flip the direction of the shaft axis
+  void FlipShaftDirection();
```

Comment:
It would be also helpful to add a few sentences to the module help text about the two types of calibration and in what order they can or should be performed.